app/src/main/java/com/goldenraven/padawanwallet/ui/drawer/SendCoinsBackScreen.kt

```diff
@@ -62,6 +62,7 @@ internal fun SendCoinsBackScreen(navController: NavController) {
 
     Scaffold(
         topBar = { DrawerAppBar(navController, title = "Send Coins Back") },
+        bottomBar = { SnackbarHost(hostState = snackbarHostState) }
```

Comment:
Somehow this solved the issue of the toast we had a few weeks back! 🔥

Reply:
Sounds good! Really not a big deal here because it's so simple but just thought I'd mention it.